integration/ide/bsp-server/resources/snapshots/build-targets-jvm-test-environments.json

```diff
@@ -18,8 +18,8 @@
         "file:///workspace/app/test/compile-resources",
         "file:///workspace/app/test/resources",
         "file:///workspace/out/app/test/compile.dest/classes",
-        "file:///coursier-cache/https/repo1.maven.org/maven2/org/scala-sbt/test-interface/1.0/test-interface-1.0.jar",
-        "file:///mill-workspace/out/libs/testrunner/entrypoint/compile.dest/classes"
+        "file:///Users/lihaoyi/.ivy2/local/com.lihaoyi/mill-libs-testrunner-entrypoint/SNAPSHOT/jars/mill-libs-testrunner-entrypoint.jar",
```

Comment:
This path doesn't look reproducible

Reply:
Should be fixed in the merged PR, I added a new `file://user-home` prefix substitution